### PR TITLE
Set the Blackbird and Talos II baseboard details again

### PR DESCRIPTION
**Pull request by hughsie**
This regressed in 2b0f925.

Fixes https://github.com/fwupd/fwupd/issues/7437

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
@meklort

File: libfwupdplugin/fu-hwids-fdt.c [hunk 1]
```diff
@@ -92,6 +93,20 @@ fu_hwids_fdt_setup(FuContext *ctx, FuHwids *self, GError **error)
 		fu_fdt_image_get_attr_str(FU_FDT_IMAGE(fdt_img), "version", &version, NULL);
 		fu_hwids_add_value(self, FU_HWIDS_KEY_BIOS_VERSION, version);
 	}
+	fdt_img_backplane =
+	    fu_fdt_firmware_get_image_by_path(FU_FDT_FIRMWARE(fdt),
+					      "vpd/root-node-vpd@a000/enclosure@1e00/backplane@800",
+					      NULL);
+	if (fdt_img_backplane != NULL) {
```
Comment (meklort) [inline libfwupdplugin/fu-hwids-fdt.c:100]: This is always returning null

Note that the `fu_fdt_firmware_get_image_by_path(FU_FDT_FIRMWARE(fdt), "ibm,firmware-versions", NULL);` line above is also returning NULL on the blackbird with both the copy I built and the fwupd version in apt.

Comment (hughsie) [reply libfwupdplugin/fu-hwids-fdt.c:100]: Hmm, can you do a `sudo ./src/fwupdtool hwids -vv` log please.

Comment (meklort) [reply libfwupdplugin/fu-hwids-fdt.c:100]: The bianry is located at `/home/meklort/git/fwupd/venv/bin/fwupdtool` following the latest readme, but...
[hwids.txt](https://github.com/user-attachments/files/16155392/hwids.txt)

Comment (hughsie) [reply libfwupdplugin/fu-hwids-fdt.c]: Aha, a missing prefix slash. Can you try the branch again please.

**Comment by meklort** [comment]
This looks good now
```Computer Information
--------------------
Manufacturer: IBM
Family: rcs,blackbird
ProductName: C1P9S01 REV 1.01
BaseboardManufacturer: RAPTOR COMP SYS
BaseboardProduct: C1P9S01 REV 1.01
Hardware IDs
------------
not available as 'ProductSku' unknown
not available as 'BiosVendor' unknown
not available as 'BiosVendor' unknown
not available as 'ProductSku' unknown
not available as 'ProductSku' unknown
{300ac84b-90f8-5517-a122-6cf0d0295136}   <- Manufacturer + Family + ProductName
not available as 'ProductSku' unknown
not available as 'ProductSku' unknown
{8f1436fe-f9b0-550a-9fe1-e1875222ba6c}   <- Manufacturer + ProductName + BaseboardManufacturer + BaseboardProduct
{cfd2e5e4-e0d2-552a-b0fe-9e57f6b7a775}   <- Manufacturer + ProductName
{9153212f-20c7-5457-a287-22e08f79ea0e}   <- Manufacturer + Family + BaseboardManufacturer + BaseboardProduct
{baa10fed-3b1a-52c6-a844-e8fd71053f62}   <- Manufacturer + Family
not available as 'EnclosureKind' unknown
{b9e02d8f-594c-597f-9f2a-6ee94f52c9ed}   <- Manufacturer + BaseboardManufacturer + BaseboardProduct
{1aa9dce5-8503-506f-a042-4493b78e92ba}   <- Manufacturer
Extra Hardware IDs
------------------
not available as 'ProductSku' unknown
not available as 'BiosVendor' unknown
not available as 'BiosVendor' unknown```

**Review by meklort — APPROVED** [review]

**Pull request merged**